src/DataFramesMeta.jl

```diff
@@ -690,6 +705,18 @@ end
 ##
 ##############################################################################
 
+function select!(d::Union{AbstractDataFrame, AbstractDict}; kwargs...)
+    tokeep = [k for (k, v) in kwargs]
```

Comment:
Could use a generator. Also don't we support integer column indices in addition to names?

Reply:
Currently they are separate functions? there is a `select(d, args)` on its own somewhere. This method is for `@select(df, :a, :b, :z = :a + :b)`, which personally i wouldn't mind deprecating in favor of putting that variable creation in `@transform` only.

Reply:
I just wondered whether `k` could be an integer here, resulting from e.g. `@select(df, 1, 2)`.

Reply:
I just looked into this and fixing it would probably a useful separate PR

@select(df, ...) takes you to:

1. `select(df, arg) = df[arg]` which works with only one integer
2. `select(df; kwargs...)`. In the process, if there is more than one argument, like `@select(df, :a, :b)` it gets transformed to `select(df, a = :a, b = :b)`. This doesn't work with integers at all. so `@select(df, 1, 2)` does not work. But the error message is not particularly insightful. It makes me think that the keyword argument creation step is failing silently. 

```
ERROR: MethodError: no method matching select(::DataFrame, ::Int64, ::Int64)
Closest candidates are:
  select(::AbstractDataFrame, ::Any) at C:\Users\Pdeffebach\.julia\dev\DataFramesMeta\src\DataFramesMeta.jl:292
  select(::Union{AbstractDataFrame, AbstractDict}; kwargs...) at C:\Users\Pdeffebach\.julia\dev\DataFramesMeta\src\DataFramesMeta.jl:722
Stacktrace:
 [1] top-level scope at C:\Users\Pdeffebach\.julia\dev\DataFramesMeta\src\DataFramesMeta.jl:752
```